modules/end-user-portal/src/Models/SearchResult.cs

```diff
@@ -1,26 +1,12 @@
-using GpConnect.NationalDataSharingPortal.EndUserPortal.Resources;
-using System.ComponentModel.DataAnnotations;
+using Newtonsoft.Json;
 
 namespace GpConnect.NationalDataSharingPortal.EndUserPortal.Models;
 
 public class SearchResult
 {
+    [JsonProperty("results")]
```

Comment:
I'll change this in the API because the property is called something else.

Reply:
thanks